robot_framework/sap.py

```diff
@@ -0,0 +1,104 @@
+"""This module handles the relevant cases in SAP"""
+
+from OpenOrchestrator.orchestrator_connection.connection import OrchestratorConnection
+
+
+def open_worklist(session):
+    """
+    Opens the worklist in the SAP application.
+
+    This function navigates to the worklist screen by setting the transaction code to 'FPCRPO'
+    and pressing the search button after clearing the 'maks. antal træffere' field.
+
+    Parameters:
+    session (SapGuiSession): The active SAP GUI session to interact with.
+    """
+    session.findById("wnd[0]/tbar[0]/okcd").text = "FPCRPO"
+    session.findById("wnd[0]").sendVKey(0)
+    # Remove the number from "maks. antal træffere" by writing an empty string
+    session.findById("wnd[0]/usr/txt%%DYN001-LOW").text = ""
+    # Press the search button
+    session.findById("wnd[0]/tbar[1]/btn[8]").press()
+
+
+def filter_searches(session):
+    """
+    Filters the search results in the SAP application.
+
+    This function filters the search results by clicking the 'Selekter' button
+    and selecting the 'Oereafrunding' option.
+
+    Parameters:
+    session (SapGuiSession): The active SAP GUI session to interact with.
+    """
+    # Press the 'Selekter' button
+    session.findById("wnd[0]/tbar[1]/btn[33]").press()
+    # Press Oereafrunding
+    session.findById("wnd[1]/usr/ssubD0500_SUBSCREEN:SAPLSLVC_DIALOG:0501/cntlG51_CONTAINER/shellcont/shell").setCurrentCell(9, "TEXT")
```

Comment:
This is hardcoded to select row 9 in the layout table. This might be a problem if the table changes.